packages/ia-zendesk-help-widget/src/ia-zendesk-help-widget.js

```diff
@@ -0,0 +1,141 @@
+import { html, css, LitElement } from 'lit-element';
+import loadingIcon from './assets/icon-loading';
+import questionIcon from './assets/icon-question-mark';
+import { LazyLoaderService } from '@internetarchive/lazy-loader-service';
+
+class ZenDeskHelpWidget extends LitElement {
+  static get styles() {
```

Comment:
I find it useful to group the `styles` method and the `render` method since it's those two things that are interacting the most. Also, since the `styles` section is so much bigger than the rest, I like putting it at the bottom to not obscure all the business logic, ie:
 
```
class ZenDeskHelpWidget extends LitElement {
  ... business logic ...
  render() { }
  styles() { }
}
```

Reply:
Putting the styles in a separate file also helps to not obscure all of the business logic, which is the primary reason why I separate them.

Reply:
That also misses the first point:
> I find it useful to group the styles method and the render method since it's those two things that are interacting the most.

Reply:
Instead of putting them in same file, we should consider styles in separate files as it was earlier since CSS code is bigger part.